### PR TITLE
Add Tooltip to mailcow Apps

### PR DESCRIPTION
**Pull request by PhoenixPeca**
Basically, this allows the admin to set an optional custom description of a specific mailcow app that can be read on mouse hover. Additionally, I have also added `htmlspecialchars` to escape double quotes in some html attributes. The descriptions are in english by default, but users can always customize them in `vars.local.inc.php` so I don't think we'll need a dedicated translation entries in the `lang` folder.
sdgdfg

### CODE REVIEW

File: data/web/inc/vars.inc.php [hunk 1]
```diff
@@ -42,11 +42,13 @@
 $MAILCOW_APPS = array(
   array(
     'name' => 'SOGo',
-    'link' => '/SOGo/'
+    'link' => '/SOGo/',
+    'description' => 'SOGo is a collaborative software (groupware) server.'
```
Comment (mkuron) [inline data/web/inc/vars.inc.php]: Change the descriptions to something less technical, e.g. "Access your email, contacts and calendar" and "Access your email".

**Review by mkuron — CHANGES_REQUESTED** [review]

File: data/web/inc/vars.inc.php (hunk 1 above)
Comment (PhoenixPeca) [reply data/web/inc/vars.inc.php]: @mkuron how about this one "SOGo is a multi-purpose groupware server that provides native Microsoft ActiveSync support."? 😄

Comment (mkuron) [reply data/web/inc/vars.inc.php]: Remember, this page is used by the end user, not that much by the admin. They might never have even heard about "groupware" or "ActiveSync". Also, ActiveSync is not used in the browser through the link here, so it doesn't need to be mentioned in the description.

Comment (PhoenixPeca) [reply data/web/inc/vars.inc.php]: Any suggestions? 😄

Comment (PhoenixPeca) [reply data/web/inc/vars.inc.php]: How about this?

"SOGo is a web-based client for email, address book and calendar."
"Roundcube is a light-weight alternative of SOGo webmail."

Comment (mkuron) [reply data/web/inc/vars.inc.php]: The first one is good. Change the second one to "Roundcube is a web-based email client" (i.e. drop the SOGo reference and don't claim it's lightweight -- it's a pretty massive piece of PHP and JS after all).

**Pull request merged**